Microsoft.Toolkit.Uwp.SampleApp/SamplePages/BackgroundTaskHelper/TestBackgroundTask.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using Microsoft.Toolkit.Uwp.Notifications;
+using Windows.ApplicationModel.Background;
+using Windows.UI.Notifications;
+
+namespace Microsoft.Toolkit.Uwp.SampleApp.SamplePages
+{
+    internal class TestBackgroundTask : IBackgroundTask
+    {
+        public void Run(IBackgroundTaskInstance taskInstance)
+        {
```

Comment:
It use it by default then, it works fine but we need a way to distinguish between both in the sample App because I use the sample app as my reference not the documentation.
I mean they should be 2 buttons on for SPM and one for MPM :) and to have a link to an article saying what is MPM and what is SPM